src/repository/todo.repository/saveToDo.ts

```diff
@@ -0,0 +1,25 @@
+import fs from "fs";
+import getToDos from "./getToDos";
+
+type ToDO = {
```

Comment:
Types should be extracted to a separate application layer. Think about database-level (Entities) and presentation-level (DTOs) types. How are they different?